### ISSUE TITLE
Alt Text Missing on "Collections" and "Works" Icons Links when Dashboard is Resized

### ISSUE THREAD

**Issue by posixeleni** [samvera/hyku#3085]
https://vpat.hykudemo.org
Version 7.1

### Descriptive summary

When the dashboard page is resized and the links on the navigation bar turn to just icons, most of the icon links have a text alternative when you hover over them, with the exception for "Collections" and "Works".  These will need to be added to ensure accessibility to the following standards:

WCAG 2.2 A: [2.4.4: Link Purpose in Context](https://www.w3.org/WAI/WCAG22/Understanding/link-purpose-in-context)
WCAG 2.2 AAA: [2.4.9: Link Purpose (Link Only)](https://www.w3.org/WAI/WCAG22/Understanding/link-purpose-link-only)
WCAG 2.2 A: [4.1.2: Name, Role, Value](https://www.w3.org/WAI/WCAG22/Understanding/name-role-value)

### Rationale
Accessibility issue remediation

### Expected behavior
Alt text should show up when hovering over the icons for **Collections** and **Works** when the Dashboard page is resized and the text is replaced by icons. 

### Actual behavior
No alt text shows up when you hover over the icons for *Collections** and **Works** when the Dashboard page is resized and the text is replaced by icons. All the other icons have alt text.

### Steps to reproduce the behavior
1. Go to Admin Dashboard: https://vpat.hykudemo.org/dashboard?locale=en
2. Resize the page either by increasing Zoom level or dragging in the browser window until the text on the left-hand navigation bar turns to icons.
3. Load SiteImprove plugin (see screenshot below) and hover over the icons for "Collections" (folder icon) and "Works" (single page icon)
<img width="1032" height="954" alt="Image" src="https://github.com/user-attachments/assets/4fd152cc-f129-4747-bb0f-ef0c35d0da6e" />

### Related work
